galactic/distribution.yaml

```diff
@@ -1215,6 +1215,21 @@ repositories:
       url: https://github.com/swri-robotics/marti_messages.git
       version: dashing-devel
     status: developed
+  mavlink:
+    doc:
+      type: git
+      url: https://github.com/mavlink/mavlink-gbp-release.git
```

Comment:
I just realized that this is pretty odd; we usually have this point to the *source* repository, not the release repository.

It actually ends up working because that particular branch of the release repository has a clone of the source, but that is not the place you want to point people to make changes.  So I'd suggest changing this to the source repository.

(all of this goes for the source entry down below as well, and also goes for the release in Foxy as well)

Reply:
Upstream is not ROS package, so it do not have package.xml.
Because of that ROS1 tools cannot build docs or properly add to source index.
Is it changed for ROS2?

Reply:
Let's try to use upstream for source and docs.

Reply:
Oh, I see.  I'm sorry, I didn't realize that's how it was setup.

In that case, I think what you had before was better.  We'll run into the same trouble with ROS 2 if there is no package.xml, as we have no way to figure out where a package begins.